src/index.tsx

```diff
@@ -24,11 +18,15 @@ export default function Command() {
       searchBarPlaceholder="Search WordPress code reference..."
       throttle
     >
-      <List.Section title="Results" subtitle={data?.length + ""}>
-        {data?.map((searchResult: SearchResult) => (
-          <SearchListItem key={searchResult.name} searchResult={searchResult} />
-        ))}
-      </List.Section>
+      {searchText.length > 0 ? (
+        <List.Section title="Results" subtitle={data?.length + ""}>
+          {data?.map((searchResult: SearchResult) => (
+            <SearchListItem key={searchResult.name} searchResult={searchResult} />
+          ))}
+        </List.Section>
+      ) : (
+        <List.EmptyView title={"Type to search"} icon={{ source: "command-icon.png" }} />
+      )}
```

Comment:
Can you a screenshot of how it looks when no query is typed, and when no results where found?

Reply:
https://user-images.githubusercontent.com/1374472/231369986-ddf3e095-f6ac-46cb-a217-7ca7e2408166.mov

Reply:
@tillkruss Here is short screen recording to show how it works

Reply:
Do I also need using `List.EmptyView` when search query returns no results?

Reply:
> Do I also need using `List.EmptyView` when search query returns no results?

No, but let's remove the WordPress icon before a query was typed, it's too big.

Reply:
Thanks for asking. No this looks fantastic!